wire-ios-data-model/Tests/Source/ManagedObjectContext/CoreDataMigrationActionFactoryTests.swift

```diff
@@ -23,6 +23,7 @@ final class CoreDataMigrationActionFactoryTests: XCTestCase {
 
     // add version with actions here - aka custom migration
     let excludedVersions: [CoreDataMessagingMigrationVersion] = [
+        .version2_116,
```

Comment:
The idea here was to write additional tests for the excluded versions... maybe it's not clear from the description?

E.g.
```swift
    func test_ItReturnsPostActionForVersion2114() {
        // given
        // when
        let action = CoreDataMigrationActionFactory.createPostMigrationAction(for: .version2_114)
        // then
        XCTAssertTrue(action is OneOnOneConversationMigrationAction)
    }
```

Reply:
Done